core/server/translations/en.json

```diff
@@ -357,7 +357,8 @@
             },
             "redirects": {
                 "missingFile": "Please select a JSON file.",
-                "invalidFile": "Please select a valid JSON file to import."
+                "invalidFile": "Please select a valid JSON file to import.",
+                "yamlParse": "The parsed result of the YAML file is a plain string. Please check the format of your redirects file."
```

Comment:
@peterzimon this needs a copy review from you. The context of the error is available here - https://github.com/TryGhost/Ghost/pull/12187/commits/9ddfdcdecdfeb5e75273f5f87a2ec460a36cf3fe#diff-48644be82a9b957e5e627bf7b0f2f73cdb1d63851ffad68c7c178c5886495bb8R48-R51. It might happen when user upload a `yaml` file which is just a "string" without any specific data (incorrect format).

Reply:
@justin-ghost ping on the above

Reply:
@naz `invalidFile` copy looks good. For `yamlParse`, here are two options to consider based on the ambiguity of the error: 

* YAML output cannot be a plain string. Check the format of your YAML file. 
* Incorrect file format. Check the format of your YAML file.

Reply:
@justin-ghost I prefer the first one because it tells users what went wrong.

Reply:
Nice. Good call with 1st option @sainthkh . Thanks for feedback @justin-ghost :black_heart:

Reply:
After having a little more thought think the message is meant to say "YAML ~output~ **input**  cannot be a plain string". We are dealing with an **input** file here. @justin-ghost let me know if this makes sens. I will go on and merge the changes no, we could modify the copy a little in master after